meta-lxatac-software/recipes-devtools/bcu/bcu/0001-CMakeLists-do-not-use-vendored-libcurl.patch

```diff
@@ -0,0 +1,38 @@
+From ef1aaeee984ab13dab5d46c73e9b207cbdf62c55 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
+Date: Wed, 20 Dec 2023 14:37:20 +0100
+Subject: [PATCH] CMakeLists: do not use vendored libcurl
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Upstream-Status: Inappropriate [upstream ticket https://github.com/nxp-imx/bcu/issues/17]
```

Comment:
I've opened an issue in the bcu repository nxp-imx/bcu#17 and added an Upstream-Status line to the patch.